server/services/config/config.go

```diff
@@ -12,7 +12,7 @@ const (
 )
 
 type AmazonS3Config struct {
-	AccessKeyId     string
+	AccessKeyId     string //nolint:stylecheck,golint
```

Comment:
I think we can here use `AccessKeyID`, but we need to change that in the usage from `mattermost-plugin/server/plugin.go` and in `server/server/server.go`